components/edit-collective/tiers/EditTierModal.js

```diff
@@ -855,7 +884,7 @@ export function EditTierForm({ tier, collective, onClose }) {
               <ModalBody>
                 <ModalSectionContainer>
                   <EditSectionContainer>
-                    <FormFields collective={collective} values={values} />
+                    <FormFields collective={collective} tier={tier} values={values} />
```

Comment:
Why introduce `tier={tier}`? Isn't value supposed to contain the latest tier info (including the type)?

Reply:
Thanks. I've corrected this. 👍🏽